src/components/artifacts/ArtifactsView.tsx

```diff
@@ -194,7 +194,7 @@ class ArtifactsView extends React.Component<Props> {
       <Paper elevation={1}>
         <Toolbar className={classes.title}>
           <Typography variant="h6" color="inherit">
-            {this._currentPath() || 'Artifacts'}
+            {'Artifacts' + this._currentPath() == null? '' : (' in ' + this._currentPath())}
```

Comment:
So it will be `Artifacts` or `Artifacts in build/artifacts`? IMO `Artifacts` or `build/artifacts` are cleaner. 🤷‍♂

Reply:
Here's another thought: `Artifacts` or `Artifacts` (newline) `build/artifacts`